src/connections/cameras.js

```diff
@@ -496,6 +496,8 @@ class Axis {
  *
  * `controller.connections.cameras` is an object of Axis camera connections
  *
+ * @typedef {Record<typeof config.axisCameras[number], Axis>} CamerasConnection

```

Comment:
A typedef inside the jsdoc for a func seems wrong?

Reply:
How about putting it in a types.d.ts?

Reply:
It can be in the file, it should just be in a separate comment stanza IMO? But if its easier to have it all living in a types file, or a types file per connection, that works too.

Reply:
Inb4 we just move this all to TS 🤣